keylime-push-model-agent/src/main.rs

```diff
@@ -187,10 +207,10 @@ async fn send_attestation_request(
 
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None, ignore_errors = true)]
-struct Args {
+pub struct Args {
```

Comment:
Is it necessary to make the `Args` structure public? If it is used only inside the crate, maybe we could limit it to be `pub(crate)`